### DISCUSSION TITLE
My rig is ASUS ROG MAXIMUS XI HERO Z390 1151, I7-8700K, Samsung NMVe 980 Pro 2TB, Vega Frontier Edition 16GB!

### DISCUSSION THREAD

**Discussion by Godofsharing** [vovinacci/OpenCore-ASUS-ROG-MAXIMUS-XI-HERO#127] [General]
Hey Vovinacci!

My rig is ASUS ROG MAXIMUS XI HERO Z390 1151, I7-8700K, Samsung NMVe 980 Pro 2TB, Vega Frontier Edition 16GB!

I been running macOS Catalina 10.15.7 for years! I made a bootable USB from Unibeast. I haven’t been able upgrade to another OS system since! Not even direct upgrade from Updates!

TonyMac is not help. They say try a script of Terminal with Clover or OpenCore. I can NEVER get a bootable USB and they refuse to give details of how to do a step by step.

Can you guide me to make a bootable USB Ventura for my board and BIOS settings?

Aubrey

**Comment by vovinacci** [comment]
Hey @Godofsharing!

Sorry to hear that you experience such issues and to reply you quite late.

I understand your pain, and that was exactly the reason why I've switched to OpenCore myself.

It's quite late here, I'll reply you with more details tomorrow.

**Comment by vovinacci** [comment]
@Godofsharing 

In general, https://dortania.github.io is the great source of knowledge on OpenCore.
The good thing about OpenCore itself, is that you don't need any additional tools in macOS itself - all the required tools are confined in `EFI` folder.

With regards to the upgrade, I would think of installing fresh copy of Ventura.

There's an extremely good article on how to [make the macOS installer](https://dortania.github.io/OpenCore-Install-Guide/installer-guide/mac-install.html#downloading-macos-modern-os). Personally, I'm using slightly different tools, however the idea stays the same.

- Prepare USB as mention [here](https://dortania.github.io/OpenCore-Install-Guide/installer-guide/mac-install.html#setting-up-the-installer)

- In Terminal

  - Clone https://github.com/corpnewt/gibMacOS to download macOS Ventura

    ```shell
    git clone https://github.com/corpnewt/gibMacOS.git
    cd gibMacOS
    ```

  - Download latest Ventura (13.2 (22D49) as of writing)

    ```shell
    % ./gibMacOS.command
    
    Available Products:

    [ … ]

    3. macOS Ventura 13.2 (22D49)
       - 032-35706 - Added 2023-01-23 18:14:21 - 12.56 GB
    ```

  - After it is downloaded (depends on the Internet connection speed mostly), create installer application in `/Applications`

    ```bash
    open ./macOS\ Downloads/publicrelease/032-35706\ -\ 13.2\ macOS\ Ventura\ \(22D49\)/InstallAssistant.pkg
    ```

  - Copy installer to the USB. Note that the command is made for USB's formatted with the name MyVolume:

    ```bash
    sudo /Applications/Install\ macOS\ Ventura.app/Contents/Resources/createinstallmedia --volume /Volumes/MyVolume/
    ```

  - Note the disk and partition for the EFI, we would need to copy contents of this EFI folder from this repository there. In my case, it's `dev/disk6s1`. Yours most probably would be a bit different.

    ```bash
    % sudo diskutil list

    /dev/disk6 (external, physical):
       #:                       TYPE NAME                    SIZE       IDENTIFIER
       0:      GUID_partition_scheme                        *30.8 GB    disk6
       1:                        EFI EFI                     209.7 MB   disk6s1
       2:                  Apple_HFS Install macOS Ventura   30.4 GB    disk6s2
    ```

  - Mount the EFI partition

    ```bash
    sudo diskutil mount /dev/disk6s1
    ```

    It should reply with `Volume EFI on /dev/disk6s1 mounted` and you would see `/Volumes/EFI` folder

- Copy the EFI folder from this repository there. After it's done, it would look like the following

  ```bash
    % tree -d /Volumes/EFI
    /Volumes/EFI
    └── EFI
    ┆   ├── BOOT
    ┆   └── OC
    ┆   ┆   ├── ACPI
    ┆   ┆   ├── Drivers
    ┆   ┆   ├── Kexts
    ┆   ┆   │   ├── AppleALC.kext
    ┆   ┆   │   │   └── Contents
    ┆   ┆   │   │       └── MacOS
    ┆   ┆   │   ├── IntelMausi.kext
    ┆   ┆   │   │   └── Contents
    ┆   ┆   │   │       └── MacOS
    ┆   ┆   │   ├── Lilu.kext
    ┆   ┆   │   │   └── Contents
    ┆   ┆   │   │       └── MacOS
    ┆   ┆   │   ├── SMCProcessor.kext
    ┆   ┆   │   │   └── Contents
    ┆   ┆   │   │       └── MacOS
    ┆   ┆   │   ├── SMCSuperIO.kext
    ┆   ┆   │   │   └── Contents
    ┆   ┆   │   │       └── MacOS
    ┆   ┆   │   ├── USBMap.kext
    ┆   ┆   │   │   └── Contents
    ┆   ┆   │   ├── VirtualSMC.kext
    ┆   ┆   │   │   └── Contents
    ┆   ┆   │   │       └── MacOS
    ┆   ┆   │   └── WhateverGreen.kext
    ┆   ┆   │       └── Contents
    ┆   ┆   │           └── MacOS
    ┆   ┆   ├── Resources
    ┆   ┆   │   ├── Audio
    ┆   ┆   │   ├── Font
    ┆   ┆   │   ├── Image
    ┆   ┆   │   │   └── Acidanthera
    ┆   ┆   │   │       ├── Chardonnay
    ┆   ┆   │   │       ├── GoldenGate
    ┆   ┆   │   │       └── Syrah
    ┆   ┆   │   └── Label
    ┆   ┆   └── Tools
    ┆   ┆   ┆   └── memtest86

    41 directories
  ```

**Comment by vovinacci** [comment]
@Godofsharing 

Few notes:

- For the EFI folder from this repository, follow https://github.com/vovinacci/OpenCore-ASUS-ROG-MAXIMUS-XI-HERO#download-efi-folder-archive-from-repository-releases-page to replace placeholders, otherwise it won't boot.
- I've tried it myself yesterday, everything seemed to work nicely, however YMMV. Feel free to ask questions, I'll try my best to answer it.

Good luck, hope that all would help. Shout should you need any help.
